telemetry/definitions/commonDefinitions.json

```diff
@@ -2601,15 +2601,41 @@
             "unit": "Count"
         },
         {
-            "name": "awsq_thumbsUp",
+            "name": "awsq_codeGenerationThumbsUp",
             "description": "User clicked on the thumbs up button, to mention that they are satisfied",
             "unit": "Count"
         },
         {
-            "name": "awsq_thumbsDown",
+            "name": "awsq_codeGenerationThumbsDown",
             "description": "User clicked on the thumbs down button to say that they are unsatisfied",
             "unit": "Count"
         },
+        {
+            "name": "awsq_approachThumbsUp",
+            "description": "User clicked on the thumbs up button, to mention that they are satisfied",
+            "unit": "Count"
+        },
+        {
+            "name": "awsq_approachThumbsDown",
+            "description": "User clicked on the thumbs down button to say that they are unsatisfied",
+            "unit": "Count"
+        },
+        {
+            "name": "aws_codeGenerationFlow",
+            "description": "A metric to see if Code Genration flow succeeded",
```

Comment:
```suggestion
            "description": "A metric to see if Code Generation flow succeeded",
```

Reply:
higher-level metrics can do more than just track whether a flow succeeded. so the description is rather narrow. 

same for the other "flow" below.